git_info/main.py

```diff
@@ -0,0 +1,59 @@
+import click
+
+from git_summarizer import process_repo
+
+
+@click.group(chain=True)
+@click.pass_context
```

Comment:
let's simplify - remove `@click.pass_context` everywhere, main function will have several arguments and `click` option decorators